src/main/java/WayofTime/bloodmagic/item/sigil/ItemSigilPhantomBridge.java

```diff
@@ -58,32 +114,146 @@ public void onSigilUpdate(ItemStack stack, World world, EntityPlayer player, int
         int posY = playerPos.getY();
         int posZ = playerPos.getZ();
 
-        double offsetPosX = posX + EXPANSION_FACTOR * playerVelX;
-        double offsetPosZ = posZ + EXPANSION_FACTOR * playerVelZ;
-        double avgX = (posX + offsetPosX) / 2;
-        double avgZ = (posZ + offsetPosZ) / 2;
+        //Standing still, sneaking or walking with framerate drops
+        if (0 <= totalVel && totalVel < 0.1) {
+            circleTemplate7x7(posX, posY, posZ, verticalOffset, world);
+        } else //anything between the first case and being slightly faster than walking
+            //walking fairly quickly on X-axis
+            if (-0.3 < playerVelZ && playerVelZ < 0.3) {
```

Comment:
This reads very strangely. It should be `playerVelZ > -0.3 && playerVelZ < 0.3`. I understand you're trying to show a range, but it just makes it more confusing.

Same for all the other ranges done this way.